gobblin-api/src/main/java/gobblin/configuration/ConfigurationKeys.java

```diff
@@ -501,6 +501,7 @@
   /**
    * Password encryption and decryption properties.
    */
+  public static final String ENCRYPT_KEY_FS_URI = "encrypt.key.fs.uri";
```

Comment:
I am not sure we need a new configuration property. ENCRYPT_KEY_LOC can just contain a qualified path.